modules/matterhorn-scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1207,6 +1207,12 @@ private boolean isBlacklisted(String mediaPackageId, Date start, Date end, Strin
     return searchInternal(startsFrom, startsTo, endFrom, endTo, alreadyScheduledEvents);
   }
 
+  /*
+   * Returns a list of events which start and/or end within a given date range.  So you could search for things which start
+   * on Tuesday between 0900 and 1000, and end between 1500 and 1600, and you could get an event which started at Epoch and
+   * ended at 1559 on Tuesday.  This is *NOT* appropriate for conflict checking, and does not check for any edge cases.
+   * sUse checkScheduleConflicts instead.
```

Comment:
sUse > Use

Reply:
That's there that s went.  I was wondering!